### PR TITLE
Add some chapter Chinese translation

### PR DESCRIPTION
**Pull request by huaize2020**
1、Add  some Chinese translation
2、Update a few error

### CODE REVIEW

**Comment by goldbergyoni** [comment]
@huaize2020 Great improvement! Welcome aboard 🌻

@MattJin @Flouse @Miigon @musicguoke @ZhyMC Would you be able to review?

File: sections/performance/block-loop.chinese.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# 不要阻塞事件循环
+
+<br/><br/>
+
+Node 主要在通过单个线程上的实现多个队列而处理事件循环。复杂高的、大型json解析、大数组上应用逻辑、不安全的正则表达式查询和大型 IO 操作都可能导致事件循环停止。 避免CPU密集型任务，将其卸载到专用服务（例如作业服务器），或 将长任务分解为小步骤然后使用工作池，都是避免阻塞事件循环的示例。
```
Comment (MattJin) [inline sections/performance/block-loop.chinese.md:5]: 或后面多了一个空格

File: sections/security/sessions.chinese.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# 修改session中间件设置
+
+<br/><br/>
+
+
+### 一段解释
+
+许多流行的会话中间件并没有应用开箱即用的最佳实践/安全 cookie 设置。 通过减少会话劫持和会话识别等攻击的威胁，将默认设置中调整这些设置可为用户和应用程序提供更好的保护。
```
Comment (Miigon) [inline sections/security/sessions.chinese.md:8]:
```suggestion
许多流行的会话中间件并没有默认应用最佳实践以及安全的 cookie 设置。将这些默认设置进行调整，可减少会话劫持和会话识别等攻击的威胁，从而为用户和应用程序提供更好的保护。
```

File: sections/security/sessions.chinese.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+最常见的默认设置是会话`name` - 在`express-session` 中这是`connect.sid`。 攻击者可以使用此信息来识别 Web 应用程序的底层框架以及特定于模块的漏洞。 将此值更改为默认值以外的值将使确定正在使用的会话机制变得更加困难。
```
Comment (Miigon) [inline sections/security/sessions.chinese.md:10]:
```suggestion
最常见的被设置为默认值的设置项是会话的`name` - 在`express-session` 中即为`connect.sid`。 攻击者可以使用此信息来识别 Web 应用程序的底层框架以及特定于模块的漏洞。 将此值更改为默认值以外的值将使确定正在使用的会话机制变得更加困难。
```

File: sections/security/regex.chinese.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# 防止恶意RegEx让Node.js的单线程过载执行
+
+### 一段解释
+
+使用正则表达式的固有风险是解析文本和匹配指定模式所需的计算资源。对于单线程事件循环占主导地位的 Node.js 平台，像解析正则表达式模式这样的 CPU 绑定的操作将使应用程序无响应。
```
Comment (Miigon) [inline sections/security/regex.chinese.md:5]:
```suggestion
使用正则表达式的固有风险是解析文本和匹配指定模式所需的计算资源。对于单线程事件循环占主导地位的 Node.js 平台，像解析正则表达式模式这样的 CPU 操作将使应用程序无响应。
```

与"client-bound"或"server-bound"用法类似，不是 CPU 绑定的意思

File: sections/security/regex.chinese.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+console.log(saferegex(emailRegex));
+
+// instead of the regex pattern, use validator:
+const validator = require('validator');
+console.log(validator.isEmail('liran.tal@gmail.com'));
+```
+
+<br/><br/>
+
+### 引用："A vulnerable Regular Expression is known as one which applies repetition"
```
Comment (Miigon) [inline sections/security/regex.chinese.md:30]:
```suggestion
### 引用："使用了重复的正则表达式是易受攻击的正则表达式"
```

File: sections/security/regex.chinese.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+来自 Liran Tal 的书 [Essential Node.js Security](https://leanpub.com/nodejssecurity)
+> 通常，程序员会使用 RegEx 来验证从用户收到的输入是否符合预期条件。 易受攻击的正则表达式被称为将重复应用于重复捕获组的正则表达式，其中要匹配的字符串由有效匹配模式的后缀加上与捕获组不匹配的字符组成。
```
Comment (Miigon) [inline sections/security/regex.chinese.md:33]:
```suggestion
> 通常，程序员会使用 RegEx 来验证从用户收到的输入是否符合预期条件。 易受攻击的正则表达式即为将重复应用于重复捕获组的正则表达式，其中要匹配的字符串由有效匹配模式的后缀加上与捕获组不匹配的字符组成。
```

**Review by Miigon — CHANGES_REQUESTED** [review]
Reviewed! Here are the suggestions.

**Comment by goldbergyoni** [comment]
@huaize2020 I would guess this one is pending on your court?

**Pull request closed without merging**